manifests/08-openshift-imagestreams.yaml

```diff
@@ -145,6 +159,8 @@ metadata:
     include.release.openshift.io/single-node-developer: "true"
     include.release.openshift.io/single-node-production-edge: "true"
 spec:
+  lookupPolicy:
+    local: true
```

Comment:
openshift/oauth-proxy resolves to a Docker Hub image. There is a risk that an end-user is pulling the (obsolete) oauth-proxy image from Docker Hub today, and with this switch will start pulling the image from the internal registry.

https://hub.docker.com/r/openshift/oauth-proxy

Reply:
That probably would be a good thing, but let's leave that for further discussion in a separate PR.